MobileFilter/sections/specific_app.txt

```diff
@@ -853,3 +853,9 @@ c0server.jags.co.jp/jss/img/banners/$app=jp.co.jags.android.Bokumaka
 ! https://github.com/AdguardTeam/AdguardFilters/issues/89875
 ! com.gitvdemo.video
 ||cupid.ptqy.gitv.tv/mixer?$replace=/"adSlots"/"adSlots_"/,app=com.gitvdemo.video
+! https://github.com/AdguardTeam/AdguardFilters/issues/100525
+@@||imasdk.googleapis.com^$app=fr.tf1.mytf1
```

Comment:
That specific rule needs to be in whitelist.

Reply:
OK, will fix it! Thanks! :thumbsup: